AsyncDisplayKit/ASCollectionView.h

```diff
@@ -171,6 +171,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
Happy to dig in and see if there is another way to trigger this behavior.

Here is my specific use case:
- x cells displayed in an `ASViewController`
- transition to 2nd layout via pinch to zoom
- fetch is ultimately controlled by number of elements in the view * batch fetch screen count
- when the number of items displayed on the screen is changed, via a zoom or transition, that must invalidate the calculation, thereby invalidating the data required to fulfill that 3x trailing promise